creusot-contracts/src/std/convert.rs

```diff
@@ -0,0 +1,27 @@
+use crate::*;
+
+/// Specification trait for `::std::convert::From`.
+pub trait From<T>: ::std::convert::From<T> {
```

Comment:
I'm copying what `Iterator` does. But why do we reuse the same name? Another name like `FromSpec` seems clearer to me.